src/neo/IO/Caching/DataCache.cs

```diff
@@ -285,7 +285,7 @@ public TValue GetOrAdd(TKey key, Func<TValue> factory)
             }
         }
 
-        public TValue TryGet(TKey key)
+        public virtual TValue TryGet(TKey key)
```

Comment:
Any reason change to virtual. Didn't find where override.

Reply:
In UT, need to use virtual to mock TryGet().

Reply:
Please don't use `virtual` for UT purpose only.

Reply:
Ok, I'll revert it